lte/cloud/go/services/subscriberdb/obsidian/models/conversion.go

```diff
@@ -170,6 +178,12 @@ func (m *MutableSubscriber) GetAssocs() []storage.TypeAndKey {
 	var assocs []storage.TypeAndKey
 	assocs = append(assocs, m.ActivePoliciesByApn.ToTKs(string(m.ID))...)
 	assocs = append(assocs, m.ActiveApns.ToTKs()...)
+	for _, policyID := range m.ActivePolicies {
```

Comment:
Here and below: add a `ToTKs` method for the respective types, rather than explicitly constructing the slice here